content/docs/reporting/analytics-report.md

```diff
@@ -86,16 +86,22 @@ Save your report for quick access in the future and to avoid reconfiguring your
 
 #### Preset Reports
 
-Your account comes with several preset saved reports which are available from the report dropdown. All preset reports are also available through a menu by clicking the “View All Reports” button in the upper right-hand corner.
+Your account comes with several preset saved reports which are available from the sidebar. All preset reports are also available through a menu by clicking the All Reports” button at the bottom of the sidebar.
 
 ![Finding Preset Reports](media/analytics-report/preset.gif)
 
 #### Your Reports
 
-Reports that you’ve saved will also appear in the same dropdown along with preset saved reports. All saved reports are also available through a menu by clicking the “View All Reports” button in the upper right-hand corner. Saved reports are automatically shared with other users on your SparkPost account.
+All saved reports are available through a modal by clicking the "All Reports” button at the bottom of the sidebar. Saved reports are automatically shared with other users on your SparkPost account.
 
 ![Finding Saved Reports](media/analytics-report/reportselection.gif)
 
+#### Favorite Reports
+
+For quick and easy access to your reports, you can save them as favorites by clicking the dropdown menu next to the reports in the sidebar.
```

Comment:
May want to add "You can save both preset reports and reports you create as favorites."  Also - is there a limit on favorites? If so, may want to add number of reports you can favorite.  If there is no limit, I'm ok with just not mentioning a limit (leave the fact that it's unlimited undocumented) in case we add a limit later.